detekt-rules/src/test/kotlin/ExpressionBodyRuleTest.kt

```diff
@@ -114,4 +114,3 @@ class ExpressionBodyRuleTest {
         assertThat(rule.lint(code)).isEmpty()
     }
 }
-
```

Comment:
Any idea why the final new line bu was striped? but only in the detekt-rules project